src/storage/mutate/AddEdgesProcessor.cpp

```diff
@@ -87,7 +87,7 @@ std::string AddEdgesProcessor::addEdges(int64_t version, PartitionID partId,
     });
     for (auto& e : newEdges) {
         std::string val;
-        std::unique_ptr<RowReader> nReader;
+        RowReader nReader = RowReader::getEmptyRowReader();
```

Comment:
You'd better use pointer of RowReader this place. Because there are lots of check null for rowReader in the code.

Reply:
https://github.com/vesoft-inc/nebula/blob/e5cefb2e68e189181ce56f74fcc89fe3037e8bcf/src/dataman/RowReader.h#L299-L313

resolve it with CPP magic. This can really help to improve performance, the total batch computing costs reduces from 13min to 5min, for about 2.6X improvement.

Access data set from the stack is much faster than from heap, for CPU hardware cache optimization.